homeassistant/components/fibaro/__init__.py

```diff
@@ -651,7 +651,11 @@ def extra_state_attributes(self):
                     self.fibaro_device.properties.batteryLevel
                 )
             if "armed" in self.fibaro_device.properties:
-                attr[ATTR_ARMED] = self.fibaro_device.properties.armed.lower() == "true"
+                armed = self.fibaro_device.properties.armed
+                if isinstance(armed, bool):
+                    attr[ATTR_ARMED] = armed
+                elif isinstance(armed, str) and armed.lower() in ("true", "false"):
+                    attr[ATTR_ARMED] = armed.lower() == "true"
```

Comment:
We don't want attributes that goes away. Better it's set to unknown.

```suggestion
                armed = self.fibaro_device.properties.armed
                if isinstance(armed, bool):
                    attr[ATTR_ARMED] = armed
                elif isinstance(armed, str) and armed.lower() in ("true", "false"):
                    attr[ATTR_ARMED] = armed.lower() == "true"
                else:
                    attr[ATTR_ARMED] = None
```

Reply:
Ok, i merged your suggestion. Thanks for your help.